src/app/modules/main/view.nim

```diff
@@ -14,6 +14,9 @@ QtObject:
       activeSectionVariant: QVariant
       tmpCommunityId: string # shouldn't be used anywhere except in prepareCommunitySectionModuleForCommunityId/getCommunitySectionModule procs
       
+  # Forward declaration
+  proc activeSectionChanged*(self:View)
```

Comment:
@jrainville this forward declaration for a signal looks a bit weird (signal is already just a single line declaration/definition) and it's already defined in the line 79 so feel free to move it from the line 79 here to the line 18.